backend/src/src/entities/song.entity.ts

```diff
@@ -5,12 +5,15 @@ export default class SongEntity extends BaseEntity {
   duration: number;
   artist: string;
   genre: string;
+  currently_playing: boolean;
+  times_ever_played: number;
 
   constructor(data: SongEntity) {
     super(data.id || "");
     this.title = data.title;
     this.duration = data.duration;
     this.artist = data.artist;
     this.genre = data.genre;
+    this.times_ever_played = data.times_ever_played;
```

Comment:
```suggestion
    this.times_ever_played = data.times_ever_played || 0;
```
porem n sei se isso funcionaria, talvez tenha q colocar no modelo tbm. seria pra garantir q o default e 0.